sample/src/main/java/com/airwallex/paymentacceptance/PaymentCartFragment.kt

```diff
@@ -133,6 +133,15 @@ class PaymentCartFragment : Fragment() {
             }
         }
 
+    private val force3DS: Boolean
+        get() {
+            return when (Settings.force3DS) {
+                SampleApplication.instance.resources.getStringArray(R.array.array_force_3ds)[0] -> false
```

Comment:
🙈 I know it's consistent but one day, we should change this to just compare strings:
```
when (xxx) {
  "true" -> true
  "false" -> false
}
```
Or just use Boolean.fromString or String.toBool or something.

Reply:
Agree, updated to `toBoolean()`